libs/extensions/tabular/exec/src/lib/table-transformer-executor.ts

```diff
@@ -39,31 +45,48 @@ export class TableTransformerExecutor
       PrimitiveValuetypes.Transform,
     );
 
-    // check input column exists
-    const inputColumn = inputTable.getColumn(inputColumnName);
-    if (inputColumn === undefined) {
-      return R.err({
-        message: `The specified input column "${inputColumnName}" does not exist in the given table`,
-        diagnostic: {
-          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-          node: context.getProperty('inputColumn')!,
-        },
-      });
+    // check input columns exist
+    for (const inputColumnName of inputColumnNames) {
+      const inputColumn = inputTable.getColumn(inputColumnName);
+      if (inputColumn === undefined) {
+        return R.err({
+          message: `The specified input column "${inputColumnName}" does not exist in the given table`,
+          diagnostic: {
+            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+            node: context.getProperty('inputColumns')!,
```

Comment:
There is `getOrFailProperty` which allows you to omit the `!` operator. Also, you should restrict the diagnostic to the property value, so not the entire property (including its name) is underlined.

Same applies for the other diagnostics.